src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
lets make Random global

Reply:
> lets make Random global

If I'll do that, it will generate index once with creating my class ColorSupplier, and I will always have one color. 
But I need to generate a new color whith every invocation of my method.

Reply:
`private Random random = new Random();`
You do not need to create new Random() every time. You can call `nextInt()` every time.